scripts/rebase.sh

```diff
@@ -83,8 +83,10 @@ download_release() {
         commit=$(echo "${line}" | cut -d ' ' -f 3)
         if [[ "${EMBEDDED_COMPONENTS}" == *"${component}"* ]] || [[ "${LOADED_COMPONENTS}" == *"${component}"* ]] || [[ "${EMBEDDED_COMPONENT_OPERATORS}" == *"${component}"* ]]; then
             title "## Cloning ${repo} at commit ${commit}..."
-            git clone "${repo}"
```

Comment:
why not simply `git clone --depth=1` "${repo}" ?

Reply:
`git clone --depth=1 "${repo}"` will have a single commit that the `master` or `main` branch tip points to. It might not be the commit we are interested in here.

Reply:
/approve

That LGTM and drastically speeds up the cloning, thank you!

Would it make sense to add a `git config --global init.defaultBranch main`, so we don't pollute the logs with the hints about missing default branch (which at least I am getting with my git version 2.31)?

Reply:
Hope you don't mind I added this to the PR, so we can merge it.

Reply:
Right!, sorry @vfreex